game-engine/game-rules.md

```diff
@@ -257,6 +285,25 @@ Interaction:
     * BOOST_SPEED   => SPEED_STATE_1
 * If changing lane and turning into a block with the obstacle the car will still be affected.
 
+###Damage
+You spoke, we listened!
```

Comment:
I would just like to state, for the record, that I, for one, definitely did not speak. No sir. No damage feature requested by me, no way.
:p
This is going to make things interesting, to say the least.